### ISSUE TITLE
Admin notifications sender avatar

### ISSUE THREAD

**Issue by Pasley70** [iionly/faq#1]
When creating a question all admins are informed by site notifications. The sender, however - according to the logic of the function - is the first registered admin of the system. This could be misleading if the site have several admins.

Would it be better to use a neutral Avatar? Or - if possible - a Site Avatar?

see start.php, function notifyAdminNewQuestion, instead of `$admins[0] -> guid`

```
elgg_get_site_entity () -> GetGuid ()
```

or

```
elgg_get_logged_in_user_guid ()
```

**Comment by iionly** [comment]
It's not so easy to get it right as it might seem.

I've switched from site entity as sender to the first admin only with the recent release. Before that site->guid was used as sender's guid. The problem is that there is no "site avatar". Therefore, the site notification displays the "question mark" image ("image file not found" fallback) instead. The site notifcation plugin kind of expects the sender to be a user entity (the same with private messages - even slightly worse as you would get "Deleted user" as sender info because the site entity is no user entity).

As the notifications are containing kind of admin only info I had chosen the first admin as sender. The logged-in user (user who asked the question) might be a better alternative instead. I will think about it. As I will likely have to work on the FAQ plugin soon to prepare compatibility with Elgg 2.0 I hope to have decided what to do until I will make the next release.

**Comment by Pasley70** [comment]
Elgg need a Site Avatar :-) Thank you for your detailed reply!

**Comment by iionly** [comment]
I've just released version 1.9.10 of the FAQ plugin. In this version the notifications on new questions sent to admins has the user who asked the question as sender. With Elgg core lacking the option to send a notification with the site entitiy as sender this seemed the only other option.
